src/components/BookForm.js

```diff
@@ -22,7 +22,9 @@ export default function BookForm() {
     const obj = {
       category,
       title: inputValue,
```

Comment:
- [ ] Kindly fix your book removal function. When I remove a book it removes all the books. Kindly fix that. It should only remove the selected book.